packages/manager/apps/pci-rancher/src/pages/listing/index.tsx

```diff
@@ -51,7 +50,7 @@ const Listing: React.FC<ListingProps> = ({ data }) => {
           </span>
         </OsdsButton>
       </div>
-      <Datagrid data={data} />
+      {data && <TableContainer data={data} />}
```

Comment:
{data ? <TableContainer data={data} /> : <OsdsSpinner ... />}

Reply:
l will add a spinner but lets create at  ticket for for a table skeleton component